src/bridge.py

```diff
@@ -40,7 +40,8 @@ def cb_sim_dvl_depth(data):
     #DVL QUATERNION
     q_NWU_auv = np.quaternion(pose_NWU_auv.orientation.w, pose_NWU_auv.orientation.x, pose_NWU_auv.orientation.y, pose_NWU_auv.orientation.z)
     q_dvlref_dvl = q_NWU_dvlref.inverse() * q_NWU_auv * q_dvl_auv.inverse()
-    euler_dvlref_auv = quaternion.as_euler_angles(q_dvlref_dvl)
+    # euler_dvlref_auv = quaternion.as_euler_angles(q_dvlref_dvl)
```

Comment:
why transformations instead of quaternion?

Reply:
its what vasily was using. idk what order the euler angles are for the dvl, just that the default mode for the transformations library is correct